cv19ResSupportV3/V3/Boundary/Response/HelpRequestUpdateResponse.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+
+namespace cv19ResSupportV3.V3.Boundary.Response
+{
+    public class HelpRequestUpdateResponse
```

Comment:
Not sure about having this class. Could possibly reuse HelpRequestGetResponse or HelpRequestCreateResponse instead ?

Reply:
Couple of things here.. 

I wouldn't usually expect a PUT/PATCH to return the full object, only a success - (204 No Content). So I'm not entirely sure there's a need to have this object.

However, if we want the API to return the response (which we probably do in the interim for the existing frontend) then I'd say that conceptually, the API response for Updating, Creating, or Getting a Help Request is the same. It's a Help Request Boundary object.

Therefore I'd expect the Responses all to be equal - probably the same class?